### ISSUE TITLE
General Error Handling

### ISSUE THREAD

**Issue by nesimtunc** [rainlab-inc/cothings-ios#2]
Until we have a general flow of handling errors, we can just silence it here, and create a new issue for handling errors.

In that case it's better to leave a TODO comment here, and/or an issue number if possible. This would also allow merging this work without getting blocked on this.

_Originally posted by @furkanmustafa in https://github.com/rainlab-inc/cothings-ios/pull/1/files/d6d4d099e6554704ed92818350e4c683822bfd42_
